doc/api/net.md

```diff
@@ -2,9 +2,14 @@
 
 > Stability: 2 - Stable
 
-The `net` module provides you with an asynchronous network wrapper. It contains
-functions for creating both servers and clients (called streams). You can include
-this module with `require('net');`.
+The `net` module provides an asynchronous network API for creating stream-based
+servers ([`net.Server`][]) and clients ([`net.Socket`][]) that implement TCP
```

Comment:
Hmm… both ends of a connection are `net.Socket`s, so maybe `servers (…) and connections (…)` is a bit better?